src/components/Forums.js

```diff
@@ -0,0 +1,121 @@
+import CardContent from '@material-ui/core/CardContent';
+import Typography from '@material-ui/core/Typography';
+import NavigationBar from './NavigationBar';
+import Box from "@material-ui/core/Box";
+
+const { REACT_APP_API_ENDPOINT } = process.env;
+
+const Forums = () => {
+
+    const { currentUser } = useAuth();
+    const history = useHistory();
+    const [email, setEmail] = React.useState('');
+    const [userID, setUserID] = React.useState('');
+
+    const [forums, setForums] = React.useState([]);
+
+    React.useEffect(() => {
+        setEmail(currentUser.email);
+        loaduserSearchByEmail(currentUser.email);
+        //loadGetForums(); 
+    }, []);
+
+    const loaduserSearchByEmail = (email) => {
+        callApiGetuserSearchByEmail(email)
+            .then(res => {
+                var parsed = JSON.parse(res.express);
+                //console.log(parsed[0].id);
+                setUserID(parsed[0].id);
+            });
+    }
+
+    const callApiGetuserSearchByEmail = async (email) => {
+        const url = `${REACT_APP_API_ENDPOINT}/userSearchByEmail`;
+        //console.log(url);
+
+        const response = await fetch(url, {
+            method: "POST",
+            headers: {
+                "Content-Type": "application/json",
+            },
+            body: JSON.stringify({
+                email: email
+            })
+        });
+        const body = await response.json();
+        if (response.status !== 200) throw Error(body.message);
+        return body;
+    }
+
+    useEffect(() => {
+        loadGetForums();
+    }, []);
+
+    const loadGetForums = async () => {
+        try {
+            const res = await callApiGetForums();
+            const parsed = JSON.parse(res.express);
+            //console.log(parsed[0].forumTitle);
+            setForums(parsed);
+        } catch (error) {
+            console.error(error);
+        }
+    }
+
+    const callApiGetForums = async () => {
+
+        const url = `${REACT_APP_API_ENDPOINT}/getForums`;
+        console.log(url);
+
+        const response = await fetch(url, {
+            method: "GET",
+            headers: {
+                "Content-Type": "application/json",
+            }
+        });
+        const body = await response.json();
+        if (response.status !== 200) throw Error(body.message);
+        return body;
+    }
+
+    return (
+        <div id="body">
+
+            <NavigationBar></NavigationBar>
+
+            <Box sx={{ position: 'absolute', top: 100, left: "35%" }}>
+                <Typography
+                    variant="h4"
+                    gutterBottom
+                    component="div">
+                    Forums
+                </Typography>
+            </Box>
+
+            <Box sx={{ position: 'absolute', top: 150, left: "35%" }}>
+            {forums.map((event) => (
+                <Card style={{ width: '500px', marginBottom: '20px' }} key={event.id}>
```

Comment:
what are your thoughts on doubling the width of the forums? Try the following and implement it if you think its better:
```
    return (
        <div id="body">

            <NavigationBar></NavigationBar>

            <Box sx={{ position: 'absolute', top: 110, left: '50%', transform: 'translate(-50%, -50%)' }}>
                ....
            </Box>

            <Box sx={{ position: 'absolute', top: 150, left: '50%', transform: 'translateX(-50%)' }}>
                ....
            </Box>
```

the first box component changes update the title of the page to be neatly centred on the page and in the correct position horizontally. 

the second box component stretches the width to 1000px and centres them on the page. 

The results look like so:
![image](https://user-images.githubusercontent.com/57375371/220817611-8e69af36-9a73-4f46-b368-5a065ac4fcd1.png)